src/Joblify.Core/Offers/OfferService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using AutoMapper;
+using Joblify.Core.Data.Models;
+using Joblify.Core.Data.UnitOfWork;
+using System.Threading.Tasks;
+
+namespace Joblify.Core.Offers
+{
+    public class OfferService : IOfferService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IMapper _mapper;
+
+        public OfferService(IMapper mapper, IUnitOfWork unitOfWork)
+        {
+            _mapper = mapper;
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task<OfferDto> AddOfferAsync(OfferDto offerDto)
```

Comment:
Maybe AddOffer instead of AddOfferAsync?

Reply:
@Bartosz97  Could you please explain why? 😄